packages/headless/src/controllers/facets/range-facet/numeric-facet/headless-numeric-facet-options.ts

```diff
@@ -26,15 +25,85 @@ import {
   numberOfValues,
 } from '../../_common/facet-option-definitions';
 
-type OptionalFacetId = Partial<
-  Pick<AutomaticRangeFacetOptions<NumericFacetRequest>, 'facetId'>
->;
+export interface NumericFacetValueOption {
+  /**
+   * The start value of the range.
+   */
+  start: number;
 
-export type NumericFacetOptions = OptionalFacetId &
-  (
-    | Omit<AutomaticRangeFacetOptions<NumericFacetRequest>, 'facetId'>
-    | Omit<ManualRangeFacetOptions<NumericFacetRequest>, 'facetId'>
-  );
+  /**
+   * The end value of the range.
+   */
+  end: number;
+
+  /**
+   * Whether to include the `end` value in the range.
+   */
+  endInclusive: boolean;
+
+  /**
+   * The current facet value state.
+   */
+  state: FacetValueState;
+}
+
+export interface NumericFacetOptions {
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  field: string;
+
+  /**
+   * Whether the index should automatically create range values.
+   *
+   * Tip: If you set this parameter to true, you should ensure that the Use cache for numeric queries option is enabled for the Facet field in your index in order to speed up automatic range evaluation (see [Add or Edit Fields](https://docs.coveo.com/en/1982/index-content/add-or-edit-a-field)).
```

Comment:
By `Facet field`, do you mean the field to use for this facet? In other words, whatever field they pass as an option should have 'use cache for numeric queries' enabled if `generateAutomaticRanges` is true, correct?
Suggestion:
```suggestion
   * Tip: If you set this parameter to true, you should ensure that the ['Use cache for numeric queries' option](https://docs.coveo.com/en/1982/#use-cache-for-numeric-queries) is enabled for this facet's field in your index in order to speed up automatic range evaluation.
```

Reply:
I didn't write this, I copied it from [`range-facets/**/request.ts`](https://github.com/coveo/ui-kit/blob/18424ee049560a570c729222d23eb31844b99a1e/packages/headless/src/features/facets/range-facets/generic/interfaces/request.ts#L15).

I believe you're right though, I'll use your suggested doc.